### ISSUE TITLE
Always align to the right?

### ISSUE THREAD

**Issue by jcs090218** [benzanol/echo-bar.el#7]
When you print a message exceeds the frame width,

![Image 2](https://user-images.githubusercontent.com/8685505/196024533-1faacfec-259a-4203-ba60-6ce2b7fc2708.png)

Is there a way to display info on the next line? So it displays

```
-----------------------------------------------------------------
| you don't have enough space here                              |
|                                 [echo-bar text on the newline]|
-----------------------------------------------------------------
```

WDYT?

**Comment by benzanol** [comment]
I agree, that would be a good idea. I'll see if I can figure out a solution.

**Comment by benzanol** [comment]
Fixed with d0a24635da51502bdda9f5451793966fffef92a8

**Comment by jcs090218** [comment]
Confirm it works! Thanks! 🚀
